great_expectations_cloud/agent/agent.py

```diff
@@ -123,9 +128,9 @@ def _listen(self) -> None:
             if subscriber is not None:
                 subscriber.close()
 
-    def _get_current_gx_agent_version(self) -> str:
-        version: str = metadata_version(self._PYPI_GX_AGENT_PACKAGE_NAME)
-        print(f"GX Agent version: {version}")
+    @classmethod
+    def _get_current_gx_agent_version(cls) -> str:
+        version: str = metadata_version(cls._PYPI_GX_AGENT_PACKAGE_NAME)
         return version
```

Comment:
@Shinnnyshinshin I moved the print side-effect from this method to the `__init__` where it is called.